### ISSUE TITLE
ng-backend contains bad Angular syntax

### ISSUE THREAD

**Issue by CaryLandholt** [CaryLandholt/atom-ng-classify#1]
``` coffee
$httpBackend.whenGet(/^.*.(html|htm)$/).passThrough()
```

should be

``` coffee
$httpBackend.whenGET(/^.*\.(html|htm)$/).passThrough()
```

**Comment by CaryLandholt** [comment]
Fixed 2f9105862386ce81a662dfb5fb5a7c83a3daec1d

**Comment by CaryLandholt** [comment]
Oops - didn't fix it

**Comment by CaryLandholt** [comment]
Fixed 6285a4cf8be89741bf39ddecd7f27b295ef2a2c9

**Comment by CaryLandholt** [comment]
OMG - bad escaping in snippets file
